fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2584,6 +2584,14 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static String profile_info_format = "default";
 
+    /**
+     * When the session variable `enable_profile` is set to `false` and `big_query_profile_threshold` is set to 0,
+     * the amount of time taken by a load exceeds the default_big_load_profile_threshold_second,
+     * a profile is generated for that load.
+     */
+    @ConfField(mutable = true)
+    public static long default_big_load_profile_threshold_second = 300;
```

Comment:
why not use session variable like `big_query_profile_threshold`？

Reply:
we don't need to change it by session, default value should be system level.